core/src/App.svelte

```diff
@@ -105,6 +105,10 @@
   let breadcrumbsEnabled;
   let contextRequested = false;
   let loadingIndicatorTimeout;
+  let btpToolLayout = LuigiConfig.getConfigBooleanValue(
+      'settings.btpToolLayout'
+    ) && GenericHelpers.requestExperimentalFeature('btpToolLayout', true);
```

Comment:
let btpToolLayout = LuigiConfig.getConfigBooleanValue(
      'settings.btpToolLayout'
    ) && GenericHelpers.requestExperimentalFeature('btpToolLayout', **true**);
    
    here experimental is set to **TRUE**